src/map/map.ts

```diff
@@ -659,9 +665,20 @@ export class PropertiesMap {
         // ======= color axis settings
         if (this._options.hasColors()) {
             // setup initial color range (must do before setting up events)
-            const values = this._colors(0)[0] as number[];
-            const { min, max } = arrayMaxMin(values);
+            const determineColorRange = (optionMin: number, optionMax: number) => {
+                // range is provided in settings by user
+                if (optionMin !== 0 && optionMax !== 0) {
```

Comment:
I don't see why 0 is not allowed here? We could have a range of `0, 4` or `-2, 0`.

Reply:
In fact, if user didn't define them, the values are set to zeros in the initialization of `MapOptions`:
```
this.color = {
    mode: new HTMLOption('string', 'linear'),
    property: new HTMLOption('string', ''),
    min: new HTMLOption('number', 0),
    max: new HTMLOption('number', 0),
};
```
Hence I check if this is not the case
```
...
if (optionMin !== 0 && optionMax !== 0) {
...
```